BlockbookOnly/Polygon/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.7'
+services:
+    app:
+        image: gostartups/blockbookonly-polygon:latest
+        restart: always
+        ports:
+            - '9144:9144'
+        environment:
+            - RPC_HOST=node
+            - RPC_PORT=8545
+        volumes:
+            - ~/blockbook/ETH/data:/home/blockbook/data
+
+    node:
+        image: gostartups/polygon-core:latest
+        restart: always
+        ports:
+            - '8545:8545'
```

Comment:
You should check on the other places what is the current config and what are the selected ports so they don't overlap.